src/main/webapp/profile_edit.jsp

```diff
@@ -1,13 +1,22 @@
 <%@page contentType="text/html" pageEncoding="UTF-8"%>
```

Comment:
The same comment as the before. Maybe there is a way to remove duplicate code and to create a template for both pages.

Reply:
Same as above